src/main/java/de/zalando/aruha/nakadi/Application.java

```diff
@@ -5,7 +5,13 @@
 
 @SpringBootApplication
 public class Application {
+
     public static void main(final String[] args) {
-        SpringApplication.run(Application.class, args);
+        FlowIdUtils.push(FlowIdUtils.generateFlowId());
```

Comment:
I don't know. Maybe not since it's already another thread that is no worker thread. I can remove it, don't care.